packages/base/lua/Utils.lua

```diff
@@ -475,12 +475,10 @@ function forEachConnection(agent, name, _sof_)
 		customError("Agent does not have a SocialNetwork named '"..name.."'.")
 	end
 
-	for mname, connection in pairs(socialnetwork.connections) do
+	return forEachOrderedElement(socialnetwork.connections, function(mname, connection)
```

Comment:
I don't know if it is good running through the connections always in the same order. Is it ok?

Reply:
forEachCell, forEachNeighbor, and forEachAgent always execute in the same order.